packages/web/src/index.ts

```diff
@@ -19,6 +19,8 @@ export {
   CellSuccessProps,
 } from './components/createCell'
 
-// TODO: Remove these in v.10, people can import from `@redwoodjs/web/toast`
-// deprecated
-export { useFlash, Flash } from './flash'
+export * from './components/RedwoodProvider'
+
+export * from './components/MetaTags'
+export { Helmet as Head, Helmet } from 'react-helmet-async'
+export { HelmetProvider } from 'react-helmet-async'
```

Comment:
Are you exporting Head, Helmet and HelmetProvider here because other people might want to use those instead of `Head`? My suggestion would be to just export `Head` and abstract away Helmet.

Reply:
Yeah - my idea was to officially document `<Head>` but given the popularity of Helmet, I thought maybe users might prefer to use that name instead. Doesn’t hurt right?

Can remove HelmetProvider though, not really sure what I was thinking there.